ts/src/privacy/auth.ts

```diff
@@ -0,0 +1,53 @@
+import { PublicKey } from "@solana/web3.js";
+import axios from "axios";
```

Comment:
Using axios (and adding it as a dependency) seems unjustified here. I would use the simpler built-in fetch